site/en/docs/extensions/mv3/user_interface/index.md

```diff
@@ -118,17 +152,53 @@ Register icons in the manifest under the `"icons"` field.
 
 ## Additional UI features {: #additional_features }
 
+### Action badge {: #badge }
+
+Badges display a colored banner on top of the action icon. They can only be used when the `"action"`
```

Comment:
Glad you asked! We should always strive to reduce friction for our devs 👍 😀 

In this case, It has to be in quotes because it's a JSON file key in the manifest.

Reply:
Glad you asked! We should always strive to reduce friction for our devs 👍 😀 

In this case, It has to be in quotes because it's a JSON file key in the manifest.